pliers/extractors/text.py

```diff
@@ -402,3 +400,138 @@ def _extract(self, stim):
+
+    ''' Uses transformers library to extract contextualized encodings for words
+    or text sequences using pre-trained BERT models.
+
+    Args:
+        pretrained_model_or_path (str): A string specifying which BERT
+            model to use. Can be one of pretrained BERT models listed at
+            https://huggingface.co/transformers/pretrained_models.html
+            or path to custom model.
+        tokenizer (str): Type of tokenization used in the tokenization step.
+            If different from model, out-of-vocabulary tokens may be treated as
+            unknown tokens.
+        framework (str): name deep learning framework to use. Must be 'pt'
+            (PyTorch) or 'tensorflow'. Defaults to 'pt'.
+        encoding_level (str): A string specifying whether encodings for each
+            token or sequence encodings are to be returned. Must be one of
+            'token', 'sequence'.
+        pooling (str): Optional argument, relevant for sequence-level
+            embeddings only. If None and encoding_level='sequence', encodings
+            for [CLS] tokens are returned. If encoding_level='sequence' and
+            numpy function is specified, token-level embeddings are pooled
+            according to specified method (e.g. 'mean', 'max', 'min').
+        model_kwargs (dict): Named arguments for pretrained model.
+            See: https://huggingface.co/transformers/main_classes/model.html
+            and https://huggingface.co/transformers/model_doc/bert.html for
+            further info.
+        tokenizer_kwargs (dict): Named arguments for tokenizer.
+            See https://huggingface.co/transformers/main_classes/tokenizer.html
+            for further info.
+    '''
+
+    _log_attributes = ('pretrained_model', 'encoding_level',
+                       'pooling', 'framework', 'tokenizer_type',
+                       'model_kwargs', 'tokenizer_kwargs')
+
+    def __init__(self,
+                 pretrained_model_or_path='bert-base-uncased',
+                 tokenizer='bert-base-uncased',
+                 framework='pt',
+                 encoding_level='token',
+                 pooling=None,
+                 model_kwargs=None,
+                 tokenizer_kwargs=None):
+
+        if framework not in ['pt', 'tf']:
+            raise(ValueError('''Invalid framework;
+                must be one of 'pt' (pytorch) or 'tf' (tensorflow)'''))
+
+        if model_kwargs is None:
+            model_kwargs = {}
+
+        if tokenizer_kwargs is None:
+            tokenizer_kwargs = {}
+
+        self.pretrained_model = pretrained_model_or_path
+        self.tokenizer_type = tokenizer
+        self.framework = framework
+        self.encoding_level = encoding_level
+        self.pooling = pooling
+        self.model_kwargs = model_kwargs
+        self.tokenizer_kwargs = tokenizer_kwargs
+        model_name = 'BertModel' if self.framework == 'pt' else 'TFBertModel'
+
```

Comment:
```suggestion
verify_dependencies(['transformers'])
```

I would add a check that transformers was imported. Otherwise you get a cryptic error because `transformers` = None